style/header.scss

```diff
@@ -0,0 +1,11 @@
+.main-header {
+  height: 50px;
+  padding: 15px;
+  background-color: $bgwhite;
+  box-shadow: $shadow;
+  box-sizing: border-box;
+
+  &__menu-toggle {
+    display: inline-block;
```

Comment:
Why is it needed? `&__menu-toggle` is an `<a>` tag and it is inline by default. I think this can be removed